environ/process/asset_pricing/double_sorting.py

```diff
@@ -107,15 +101,26 @@ def _asset_pricing_preprocess(
     # lag 1 unit for the dominance var and yield var to avoid information leakage
     df_panel = lag_variable_columns(
         data=df_panel,
-        variable=[dominance_var] + list(yield_var.values()),
+        variable=[dominance_var, yield_var],
         time_variable="Date",
         entity_variable="Token",
     )
 
-    # split the series into stablecoin and nonstablecoin
-    df_panel["is_stable"] = df_panel["Token"].isin(STABLE_DICT.keys())
+    return df_panel
+
+
+def stable_nonstable_split(
```

Comment:
i don't think this function is particularly necessary

if i need to run a script with only stable coin I'd just say something like
`panel_to_run = df_panel[df_panel["Token"].isin(STABLE_DICT.keys())]`
before running the portfolio function

Reply:
Fixed.